specs/core/0_beacon-chain.md

```diff
@@ -185,72 +214,9 @@ Unless otherwise indicated, code appearing in `this style` is to be interpreted
 **Notes**
 
 * See a recommended min committee size of 111 [here](https://vitalik.ca/files/Ithaca201807_Sharding.pdf); the shuffling algorithm will generally ensure the committee size is at least half the target.
-* The `SQRT_E_DROP_TIME` constant is the amount of time it takes for the inactivity leak to cut deposits of non-participating [validators](#dfn-validator) by ~39.4%.
+* The `INACTIVITY_PENALTY_QUOTIENT` equals `SQRT_E_DROP_TIME**2` where `SQRT_E_DROP_TIME := 2**17 slots` (~9 days) is the amount of time it takes for the inactivity penalty to cut deposits of non-participating [validators](#dfn-validator) by ~39.4%. The portion lost by offline [validators](#dfn-validator) after `D` epochs is about `D*D/2/INACTIVITY_PENALTY_QUOTIENT`.
```

Comment:
I feel it's not very coherent when I read this. Maybe move the description of `INACTIVITY_PENALTY_QUOTIENT` to "Constants - Reward and penalty quotients" section?

Reply:
Right. Each constant section can have their own notes 👍Will fix.